dpgen/auto_test/Interstitial.py

```diff
@@ -210,7 +215,7 @@ def make_confs(self,
                     with open(insert_element_task, 'a+') as fout:
                         print(self.insert_ele[0], file=fout)
                     dumpfn(self.supercell, 'supercell.json')
-                    pos_line[-2] = '%.6f' % float(latt_param/4/super_latt_param) + ' ' + '%.6f' % float(latt_param/2/super_latt_param) + ' 0.000000 ' + self.insert_ele[0]
+                    pos_line[chl] = '%.6f' % float(latt_param/4/super_latt_param) + ' ' + '%.6f' % float(latt_param/2/super_latt_param) + ' 0.000000 ' + self.insert_ele[0]
```

Comment:
Could you please explain this change? obviously chl != -2

Reply:
The variable chl is defined at line 195, which specifies the insertion atom line position in the POSCAR to be changed. This number is different from that of the previous version of Pymatgen.